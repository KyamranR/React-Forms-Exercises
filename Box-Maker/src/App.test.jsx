import { render } from "@testing-library/react";
import App from "./App";
import "@testing-library/jest-dom";

test("renders without crashing", () => {
  render(<App />);
});

test("matches snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
