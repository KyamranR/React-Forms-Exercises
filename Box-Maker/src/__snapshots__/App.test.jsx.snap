// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Color Box Maker
    </h1>
    <div>
      <form>
        <label
          for="width"
        >
          Width:
        </label>
        <input
          id="width"
          name="width"
          type="text"
          value=""
        />
        <label
          for="height"
        >
          Height:
        </label>
        <input
          id="height"
          name="height"
          type="text"
          value=""
        />
        <label
          for="backgroundColor"
        >
          Background Color:
        </label>
        <input
          id="backgroundColor"
          name="backgroundColor"
          type="color"
          value="#000000"
        />
        <button>
          Add Box
        </button>
      </form>
      <div />
    </div>
  </div>
</DocumentFragment>
`;
